.travis.yml

```diff
@@ -1,12 +1,27 @@
-git:
-  depth: 10
+language: node_js
+node_js:
+- '0.11'
+- '0.12'
+- '4.0'
+- '5.0'
+- '6.0'
+os:
+- linux
+- osx
+script: curl -s https://raw.githubusercontent.com/atom/ci/master/build-package.sh
```

Comment:
The above script uses the system `npm` to install everything for the linting, this is important as anything below Atom v1.10.0 can't install `eslint@3`. For now at least, if you are using `eslint@3` (or anything else that requires Node.js past v0.10.40), the above scripts are the workaround.